resources/metadata/vaultRotateSecretId.yaml

```diff
@@ -139,3 +140,51 @@ spec:
           - STAGES
           - STEPS
         description: The Azure DevOps pipeline ID. Also called as definition ID
+      - name: githubToken
+        aliases:
+          - name: token
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        description: GitHub personal access token, with the repo scope
+        secret: true
+        mandatoryIf:
+          - name: secretStore
+            value: ghactions
+        resourceRef:
+          - type: vaultSecret
+            name: password
```

Comment:
On other occasions this is named `githubVaultSecretName`, not sure that this means in detail.

Reply:
I thinks it's the parameter name to change the vault lookup name?!

Reply:
I think it doesn't have an effect, but I've aligned this parameter with the other `githubToken` ones now.